src/webui/components/module.ts

```diff
@@ -1,5 +1,6 @@
 
 export { default as ApplicationSidebar } from './application/sidebar/Sidebar';
+export { default as ErrorCatcher } from './application/ErrorCatcher';
```

Comment:
Good point. As discussed, it should be a common component instead. Also the name `ErrorBoundary` suits better because it's more common for this use case.